ks-frontend-services/src/service/external/sheets.ts

```diff
@@ -16,9 +16,11 @@ const sheetsPoster = <T>(
   >;
 
 export interface ReadRequest {
-  body: {
-    spreadsheet_id?: string;
+  params?: {
```

Comment:
if `params` is optional object, will it be better to have object keys in required but have sane defaults?

otherwise the `params` true-sy check is leaky and subsequently will require each object key/value checks and give default values anyway.